spec/requests/api/v1/base_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe "Base Controller", type: :request do
+  before do
+    base_controller = Class.new(Api::V1::BaseController) do
+      def index
+        render json: {message: "Successfully autenticated"}
+      end
+    end
+    stub_const("BaseController", base_controller)
+    Rails.application.routes.disable_clear_and_finalize = true
+    Rails.application.routes.draw do
+      get "/index", to: "base#index"
+    end
+  end
```

Comment:
This is okay for now, but let's remove once we have a route we can actually test authentication against.

Reply:
alright.